client/src/TagExplorer/TagExplorerComponent.js

```diff
@@ -2,14 +2,22 @@ import classNames from "classnames";
 import _ from "lodash";
 import React, { Fragment } from "react";
 
+import common_lang from "src/panels/panel_declarations/misc/key_concept_panels/common_questions.yaml";
```

Comment:
Any FAQ yaml files that are actually shared between infographics and other routes should be lifted up to a higher level in the folder tree, maybe put them in somewhere like `src/common_text/faq`.

Small nitpick, but there's a `common_lang.yaml` in `src/common_text` already, importing this as `common_lang` might be confusing at a glance. Maybe generally import the faq yaml as `<prefix>_faq` instead of `<prefix>_lang` or something like that, to differentiate them at a glance.